### DISCUSSION TITLE
SVPi-hole Gravity update + 'Unattended-update'-Upgrades bei zufällig gleichzeitiger Ausführung gefährlich?

### DISCUSSION THREAD

**Discussion by fazer1** [RPiList/specials#1270] [Q&A] [answered]
Ich habe auf meinem Raspi 4 (2 GB) das SVPi-hole nachgebildet und zusätzlich u.a. Wireguard installiert, und in diesem Zuge auch unattended upgrades. Letzteres macht zufällig zwei Mal am Tag den Check auf neue Updates. Einmal wöchentlich sonntags in der früh macht das SVPi-hole ja das Gravity-Update.
Ich frage mich, ob es problematisch wäre, wenn beide Vorgänge zufällig aufeinanderfallen würden? Wäre das zuviel und könnte das System abstürzen?

**Comment by LizenzFass78851** [comment, marked as answer]
Das sollte eigentlich kein Problem sein wenn das Gravity Update und das unattended upgrade parallel laufen würden, da das unattended Upgrade (bei debian / ubuntu) über apt läuft und das gravity Update läuft über pihole selbst und da das pihole nur den Gravity Update macht (also die Listen aktualisiert) sollte da auch nichts passieren. 
Ob das in Kombination auf einem RPi 3 Model B+ funktioniert ohne das der Pi 3 in die Knie geht, kann ich nicht sagen.
